12-reviewing_bugs.Rmd

```diff
@@ -1,41 +1,49 @@
-# Reviewing Bugs
-
-## Python triage team
-
-## Becoming a member of the Python triage team
-
-### GitHub Labels for PRs
-
-## Fields in the Bug Tracker
-
-### Title
-
-### Type
-
-### Stage
-
-### Components
-
-### Versions
-
-### Priority
-
-### Keywords
-
-### Nosy List
-
-### Assigned To
-
-### Dependencies
-
-### Superseder
-
-### Status
-
-### Resolution
-
-### Mercurial Repository
-
-## Generating Special Links in a Comment
-
-## Checklist for Triaging
+# Reviewing Bugs
+
+## How you can help to review bug reports?
+
+After understanding where bugs are reported in $\textsf{R}$ (Bugzilla) or in other projects (GitHub), a great way to contribute is reviewing bug reports.
+
+Around the clock, new bug reports are being submitted on Bugzilla or the bug trackers (for instance, GitHub issues) of $\textsf{R}$ packages and existing bug reports are being updated. Every bug report needs to be reviewed to make sure various things are in proper order. You can help with this process of reviewing bugs. 
+
+### Preparing to review bug reports
+
+1. If you want to review bug reports on Bugzilla, you are required to have a Bugzilla account. To get a Bugzilla account send an e-mail to `bug-report-request@r-project.org` from the address you want to use as your Bugzilla login. Briefly explain why you want a Bugzilla account and a volunteer will add you to $\textsf{R}$'s Bugzilla members.
+
+
+2. More details on how you can review a bug report are available on this [blog](https://developer.r-project.org/Blog/public/2019/10/09/r-can-use-your-help-reviewing-bug-reports/)
+
+### Classifying bug reports
+
+A good bug report is the one which:
+
+1. Explains clearly how to reproduce the bug.
+
+
+2. Includes the version of $\textsf{R}$, the machine architecture, and the operating system platform on which the bug occurred.
+
+Relevant details should be a part of a good bug report. You can help with the following tasks once you have some experience developing for $\textsf{R}$:

```

Comment:
experience developing for R -> R programming experience (contributors do not need experience developing for R itself)